data-processing/scripts/dsk/utils.py

```diff
@@ -0,0 +1,201 @@
+from dataclasses import dataclass
```

Comment:
This entire script is really a module, containing functions that you import elsewhere. Shouldn't it belong to `src/dfm_data` for that reason? Then you can import them in your script with `from dfm_data.utils import X`.

The same thing is really the case with all the functions in `convert_dsk_to_jsonl`, except the `crawl_directory` function.

Reply:
Moved all methods except `crawl_directory` to `dfm_data`